docs/releasing.md

```diff
@@ -35,7 +35,7 @@ release of the Gatekeeper Operator using the GitHub Actions release workflow.
    i.e. no release candidate, such that users would have the previous released
    version installed in their cluster via OLM:
     ```shell
-    sed -Ei "s/(replaces: gatekeeper-operator.)v0.1.1/\1${RELEASE_PREV_VERSION}/" ./config/manifests/bases/gatekeeper-operator.clusterserviceversion.yaml
+    sed -Ei "s/REPLACES_VERSION \?= .+/REPLACES_VERSION ?= $RELEASE_PREV_VERSION/" Makefile
```

Comment:
Since this is override-able, shouldn't this be `export REPLACES_VERSION` so that the Makefile isn't updated directly?
```suggestion
    export REPLACES_VERSION =${RELEASE_PREV_VERSION}
```

Reply:
@dhaiducek then `make bundle` will provide different results when run in CI or in downstream builds.

Reply:
Oh, I didn't realize these instructions were intended to be commited. Never mind, then!